src/rpc/net.cpp

```diff
@@ -702,7 +686,9 @@ static RPCHelpMan listbanned()
 {
     return RPCHelpMan{"listbanned",
                 "\nList all manually banned IPs/Subnets.\n",
-                {},
+                {
+                    {"ip", RPCArg::Type::STR, "all ips", "If IP is provided, only bans which include the IP will be returned."},
```

Comment:
`RPCArg::Optional::OMITTED_NAMED_ARG`?

Reply:
Thanks for showing me that. Much better.